### ISSUE TITLE
masurca 4.0.9 reports version as 4.0.5

### ISSUE THREAD

**Issue by stetner** [alekseyzimin/masurca#283]
In the mascura 4.0.9 code:

~/90days/build/MaSuRCA-4.0.9

grep '4.0.5' */*/*/* 2>/dev/null
...
global-1/SuperReads/src/masurca:my $version_str="version 4.0.5";
...

So after build 0f 4.0.9 source:

$ masurca --version
version 4.0.5

**Comment by gvic628** [comment]
Hi stetner!
The same thing is happening to me. I installed masurca v4.0.9 on linux with this command: https://anaconda.org/bioconda/masurca. When I masurca --version, I get version 4.0.5... were you able to sort this issue out?

**Comment by stetner** [comment]
No, just need to update that string I assume, but not a big enough issue for me to worry about.  Just flagging it for the maintainers for next build.
